src/main/java/org/apache/accumulo/examples/client/TracingExample.java

```diff
@@ -75,7 +75,7 @@ private TracingExample(AccumuloClient client) {
   }
 
   private void enableTracing() {
-    // DistributedTrace.enable("myHost", "myApp");
+    TraceUtil.enable();
```

Comment:
> because this is an example for end user code, we should avoid TraceUtil, and make calls directly to OpenTelemetry, as needed

I think I misunderstood the usage of this example. I don't think we should maintain an example for how to use OpenTelemetry. There is better documentation and better examples at https://opentelemetry.io/docs/instrumentation/java/. I think we should remove this example.

Reply:
I have already updated the website tracing documentation for [2.1.0](https://github.com/apache/accumulo-website/pull/265/files#diff-ce670c301bef1b4fc4ca07091937f291f385b3c5c0163542260ffe9f4351103b)